src/realm/sync/noinst/client_reset_recovery.cpp

```diff
@@ -626,52 +626,68 @@ void RecoverLocalChangesetsHandler::copy_lists_with_unrecoverable_changes()
 bool RecoverLocalChangesetsHandler::resolve_path(ListPath& path, Obj remote_obj, Obj local_obj,
```

Comment:
This method is one of the trickier changes in this PR, could you take a look at the coverage report and try to test some more of the uncovered corner cases?

Reply:
agreed.